src/literal_finder.rs

```diff
@@ -20,20 +20,23 @@ impl LiteralFinder {
             found_literal: None,
         }
     }
+
+    pub fn between(&self, location: &SourceLocation) -> bool {
+        location.start <= self.cursor_location && self.cursor_location < location.end
+    }
```

Comment:
Huh, was this really automatically added by clippy?

Reply:
No. It was me. Clippy just complained you can merge the if statements. Maybe gpt-clippy one day

Reply:
Haha, looked to smart to be true :)